crates/forge/tests/cli/test_cmd.rs

```diff
@@ -50,9 +50,11 @@ contract Dummy {}
     cmd.args(["test"]);
 
     // run command and assert
-    cmd.unchecked_output().stdout_matches_path(
-        PathBuf::from(env!("CARGO_MANIFEST_DIR")).join("tests/fixtures/warn_no_tests.stdout"),
-    );
+    cmd.assert().stdout_eq(str![[r#"
```

Comment:
we can also assert failure here and elsewhere; assert().failure()

Reply:
yep, added